src/FakeItEasy/Creation/CreationResult.cs

```diff
@@ -6,11 +6,14 @@ namespace FakeItEasy.Creation
     using System.Text;
     using FakeItEasy.Core;
 
-    internal abstract class CreationResult
+    internal abstract class CreationResult(int status)
     {
+        private const int SuccessfulCreation = 0;
+        private const int FailedCreation = 1;

```

Comment:
A private enum might be preferable, but I was lazy. Happy to introduce one if you like.

Reply:
Well, an enum would actually be _less_ code, not more (at least for the declaration). I think it would be slightly better, but I won't fight you for it.